src/Tizen.NUI.Components/Controls/RecyclerList/LayoutManager.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Components
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class LayoutManager
+    {
+        public enum Orientation
+        {
+            Vertical = 0,
+            Horizontal = 1,
+        }
+
+
+        protected float mPrevScrollPosition = 0.0f;
+        protected int mPrevFirstDataIndex = 0;
+
+        public View Container{get;set;}
+        public Size ItemSize{get;set;} = new Size();
+        public Orientation LayoutOrientation{get;set;} = Orientation.Vertical;
+        public float StepSize{
```

Comment:
I think it is better to unify brace format with other major NUI source code, such View.cs

```
public float StepSize
{
    get
    {
        return mStepSize;
    }
}
```